app1.js

```diff
@@ -1,7 +1,7 @@
 function getUserName() {
     let userName = prompt('Please Enter your Name');
 
-    if(userName.toLowerCase() == 'roger') {
```

Comment:
Get into the habit in JS of using '===' for strict equality comparisons, it'll save you headaches later. Also, read this: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a